piou-piou/ribs-admin-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Ribs\\RibsAdminBundle\\": ["all"]
```

Comment:
this is invalid. the bundle class name does not end with a `\`

Reply:
thanks i will fix that. I made some updates on package before to push that change

Reply:
all is fixed in last commit, you can close you review :)